examples/custom-server-actionhero/README.md

```diff
@@ -1,6 +1,6 @@
 # Running Next.JS and React /inside/ of ActionHero
 
-This server will render dynamic next.js/react pages on some routes, and normal ActionHero API requests on others.  
```

Comment:
Two spaces in markdown actually means a line break. Please replace this with a `<br>`.